aidial_adapter_vertexai/chat/gemini/adapter.py

```diff
@@ -124,34 +126,48 @@ async def parse_prompt(
             case _:
                 assert_never(self.deployment)
 
+    def _get_model(
```

Comment:
We can move this function back to `vertex_ai` module, and make it cached, what do you think?

```python
@cached
def get_generative_model(
    model_id:str,
    ...
)

```


My main reasoning behind this - I think it was pretty convenient, when all models object from vertexai were in one place